sql/plan/group_by.go

```diff
@@ -233,15 +233,35 @@ func (i *groupByIter) Next(ctx *sql.Context) (sql.Row, error) {
 		return nil, io.EOF
 	}
 
-	i.done = true
-
+	// special case for any_value
 	var err error
+	onlyAnyValue := true
 	for j, a := range i.selectedExprs {
 		i.buf[j], err = newAggregationBuffer(a)
 		if err != nil {
 			return nil, err
 		}
+		// TODO: bad, figure out better way to determine if there are any agg selections that aren't any_value
+		// maybe should be handled in analyzer?
```

Comment:
Good/helpful note. I think this is totally fine to special case this here like this for now.